src/api/cpp/cvc5.cpp

```diff
@@ -5463,6 +5518,16 @@ Sort Solver::mkBitVectorSort(uint32_t size) const
   CVC5_API_TRY_CATCH_END;
 }
 
+Sort Solver::mkFiniteFieldSort(const std::string& modulus) const
```

Comment:
Ditto (`solver_black`)

Reply:
:+1:

Reply:
Did you add this? I don't see any changes in `solver_black`.

Reply:
Whoops. I missed it. Sorry! Done now.